releasenotes/notes/2.0/update-split-2q-unitaries-with-swap-557a1252e3208257.yaml

```
@@ -3,8 +3,8 @@ features_transpiler:
   - |
     The :class:`.Split2QUnitaries` transpiler pass has been upgraded to
     handle the case where the unitary in consideration can be written
-    as a SWAP gate and two 1-qubit gates. In this case, it splits the
-    unitary and also applies virtual swapping similar to what is done in
+    as a SWAP gate and two single-qubit gates. In this case, it splits the
```

Maybe link to Swap Gate?
```suggestion
    as a :class:`.SwapGate` gate and two single-qubit gates. In this case, it splits the
```